src/SOS/dbgutil/CMakeLists.txt

```diff
@@ -18,15 +22,5 @@ if(NOT DEFINED CLR_CMAKE_HOST_OSX)
     )
 endif(NOT DEFINED CLR_CMAKE_HOST_OSX)
 
-if(CLR_CMAKE_HOST_ALPINE_LINUX)
-    add_definitions(-DTARGET_ALPINE_LINUX)
-endif(CLR_CMAKE_HOST_ALPINE_LINUX)
-
-if(CLR_CMAKE_HOST_UNIX)
-    add_definitions(-DHOST_UNIX)
-    add_compile_options(-fPIC)
```

Comment:
Any reason to remove the position independent code here? I don't think it was removed in the other libraries.

Reply:
It is set globally https://github.com/dotnet/diagnostics/blob/99fab307ee14f1e6391de73dc4e01a329c1403f9/eng/native/configureplatform.cmake#L419

Reply:
Yes, am11 is correct.

Reply:
There's plenty places where we do it manually, can we remove them then?

Reply:
I have already removed them in my next dbgshim PR.